Added docs related with usage (#604)
kubearchive/kubearchive: docs/modules/ROOT/pages/index.adoc

KubeArchive is a utility that stores Kubernetes resources off of the

> Kubernetes cluster. This enables users to delete those resources from

Reviewer: Based on the previous suggestion:
```suggestion
Kubernetes cluster. This enables users to remove those resources from
```

reasons, would benefit from KubeArchive. Another example would be users
that run build systems on top of Kubernetes (Shipwright, Tekton) that use
resources for one-shot builds and want to keep track of those builds over time.

== This documentation

> * To start with KubeArchive, go to

Reviewer 2: ```suggestion
* To get started with KubeArchive, go to
```

xref:getting-started/installation.adoc[].

> * For more advanced topics, inspect the "Configuration" and "Integrations" sections

Reviewer 2: ```suggestion
* For more advanced topics, check out the "Configuration" and "Integrations" sections
```

in the sidebar.

> * For things related to contributors, go to

Reviewer 2: ```suggestion
* For information for contributors, go to
```
